configurator/configurator-impl/src/main/java/org/codice/ddf/admin/configurator/impl/AdminOperation.java

```diff
@@ -24,14 +29,36 @@
 import javax.management.MalformedObjectNameException;
 
 import org.codice.ddf.admin.configurator.ConfiguratorException;
+import org.codice.ddf.admin.configurator.Operation;
+import org.codice.ddf.admin.configurator.Result;
+import org.codice.ddf.internal.admin.configurator.opfactory.AdminOpFactory;
 import org.codice.ddf.ui.admin.api.ConfigurationAdminMBean;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * Transactional handler for persisting bundle configuration file changes.
  */
-public class AdminOperation implements OperationBase, Operation<Void, Map<String, Object>> {
+public class AdminOperation implements Operation<Void> {
+    public static class OpFactory implements AdminOpFactory {
```

Comment:
Renamed interfaces to `*Actions` and static inner classes are now all named `Actions`.